source/guide_mumble.rst

```diff
@@ -0,0 +1,192 @@
+  HTTP request sent, awaiting response... 200 OK
+  Length: 11939824 (11M) [application/octet-stream]
+  Saving to: ‘murmur-static_x86-66.6.6.tar.bz2’
+  
+  100%[======================================================================================================================================>] 11,939,824  7.45MB/s   in 1.5s   
+  
+  2018-10-06 21:14:37 (7.45 MB/s) - ‘murmur-static_x86-66.6.6.tar.bz2’ saved [11939824/11939824]
+  
+  [isabell@stardust ~]$
+
+
+extract it and rename it to mumble:
+
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ tar xvjf murmur-static_x86-66.6.6.tar.bz2 
+  murmur-static_x86-66.6.6/
+  murmur-static_x86-66.6.6/murmur.x86
+  murmur-static_x86-66.6.6/LICENSE
+  murmur-static_x86-66.6.6/dbus/
+  murmur-static_x86-66.6.6/dbus/weblist.pl
+  murmur-static_x86-66.6.6/dbus/murmur.pl
+  murmur-static_x86-66.6.6/ice/
+  murmur-static_x86-66.6.6/ice/icedemo.php
+  murmur-static_x86-66.6.6/ice/Murmur.ice
+  murmur-static_x86-66.6.6/ice/weblist.php
+  murmur-static_x86-66.6.6/murmur.ini
+  murmur-static_x86-66.6.6/README
+  [isabell@stardust ~]$ mv murmur-static_x86-66.6.6/ mumble/
+  [isabell@stardust ~]$
+
+
+Configuration
+=============
+
+Configure port
+--------------
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ uberspace port add
+  Port 40200 will be open for TCP and UDP traffic in a few minutes.
+  [isabell@stardust ~]$
+
+
+edit config file ~/mumble/murmur.ini to specify the desired port, domain and mysql-database. Minimal config would look like this:
+
+.. code-block:: ini
+  :emphasize-lines: 1,3,4,12
+  
+  database=<username>_mumble
+  dbDriver=QMYSQL
+  dbUsername=<username>
+  dbPassword=<mysql_password>
+  dbHost=localhost
+  dbPort=3306
+  dbPrefix=murmur_
+  dbOpts="UNIX_SOCKET=/var/lib/mysql/mysql.sock"
+  dbus=session
+  icesecretwrite=
+  welcometext="<br />Welcome to this server running <b>Murmur</b>.<br />Enjoy your stay!<br />"
+  port=<your_port>
+  serverpassword=
+  bandwidth=72000
+  users=20
+  [Ice]
+  Ice.Warn.UnknownProperties=1
+  Ice.MessageSizeMax=65536
+
+Setup daemon
+------------
+
+Create ``~/etc/services.d/mumble.ini`` with the following content:
+
+.. code-block:: ini
+
+  [program:mumble]
+  command=%(ENV_HOME)s/mumble/murmur.x86 -fg -ini %(ENV_HOME)s/mumble/murmur.ini
+
+Tell ``supervisord`` to refresh its configuration and start the service:
+
+::
+
+ [isabell@stardust ~]$ supervisorctl reread
+ mumble: available
+ [isabell@stardust ~]$ supervisorctl update
+ mumble: added process group
+ [isabell@stardust ~]$ supervisorctl status
+ mumble                            RUNNING   pid 26020, uptime 0:03:14
+ [isabell@stardust ~]$
+
+If it's not in state ``RUNNING``, check your configuration.
+
+
+Finishing installation
+======================
+
+Open Mumble with your URL and the port.
+
+Best practices
+==============
+
+TODO: get an valid ssl-certificate
```

Comment:
That will be possible in the near future. For now I think it's best to move this TODO into an issue on GitHub and remove it here.

Reply:
i made the "Best practice" as a comment